lib/core/src/server/preview/to-require-context.ts

```diff
@@ -9,7 +9,7 @@ const fixBadGlob = deprecate(
     return match.input.replace(match[1], `@${match[1]}`);
   },
   dedent`
-    You're specified an invalid glob, we've attempted to fix it, please ensure the glob you specify is a valid glob. See: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-main-js
+    You're specified an invalid glob, we've attempted to fix it, please ensure the glob you specify is a valid glob. See: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs
```

Comment:
I think this link is used twice in this file? Commenting from my phone so hard to check

Reply:
I guess the phone missed the colors of the diff hehe

Reply:
aha the other bad link is in `entries.js` a different file. can you fix that too?

Reply:
done!